openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts

```diff
@@ -189,7 +189,7 @@ test.describe('Login flow should work properly', () => {
     await expect(page1.getByTestId('nav-user-name')).toContainText(/admin/i);
 
     // Wait for token expiry
-    await page2.waitForTimeout(3 * 60 * 1000);
+    await page2.waitForTimeout(2 * 61 * 1000);
```

Comment:
Shouldn't this be `2 * 60 * 1000`?

Reply:
I intentionally kept 61 to spend more time then expiry timer to be on safe side and avoid flaky tests

Reply:
In that case, it'd be good to have the explanation there for the calculations.

Reply:
Added